### ISSUE TITLE
[A11y] Desktop/Mobile Work Show Page-Single Item: NU Checker

### ISSUE THREAD

**Issue by azle** [OregonDigital/OD2#1996]
### Operating System

MacOS

### Browser

Chrome

### Affected page

Work Show

### Admin user?

No

### Issue Description

Work Show Page-Single Item: https://staging.oregondigital.org/concern/images/1g05fb620

NU Html Checker list of errors, see attached zip file.
[Work Show Page-Single Item.zip](https://github.com/OregonDigital/OD2/files/7049874/Work.Show.Page-Single.Item.zip)

### Can anybody QA or does this need an expert?

Needs expert

**Comment by straleyb** [comment]
QA: - [ ] Removed ref from wiki citation.

Other issues have been fixed or were unable to be located.

@azle
